assets/bindata.go

```diff
@@ -370,18 +370,18 @@ type bintree struct {
 }
 
 var _bintree = &bintree{nil, map[string]*bintree{
```

Comment:
We'll need to modify https://github.com/ipfs/go-ipfs/blob/884a5aebd748336db963e91daf7b916054486a99/assets/assets.go#L5 to pass `-s`.